internal/super/flix.go

```diff
@@ -218,6 +225,10 @@ func generateFlixCmd(
 ) (result command.Result, err error) {
 	cadenceFile := args[0]
 	depContracts := getDeployedContracts(state)
```

Comment:
I think you need to be pulling all contracts from state and not just deployed contracts. For example, you could be using a third party contract, have it in your `flow.json`, and not have it deployed. It would then miss grabbing it in this code.

Reply:
Good call. I added dependencies from flow.json using alias, that looks to be the format that contract dependencies puts in the flow.json. Updated the PR. thanks